apps/phystrax/metadata.json

```
@@ -0,0 +1,16 @@
+{ "id": "phystrax",
+  "name": "PhysTrax",
+  "shortName":"PhysTrax",
+  "icon": "app.png",
+  "version":"0.01",
+  "description": "Tracking physiological measurements to support active learning in classrooms",
+  "tags": "health",
+  "interface": "interface.html",
+  "readme": "README.md",
+  "supports": ["BANGLEJS2"],
+  "storage": [
+    {"name":"phystrax.app.js","url":"app.js"},
+    {"name":"phystrax.img","url":"app-icon.js","evaluate":true}
+  ],
+  "data": [{"wildcard":"heart_rate_data.?.csv"}]
```

Looks like a small issue here

> `App phystrax data file wildcard heart_rate_data.?.csv does not include app ID`

The fix being something like this (we'd also need to update `interface.html` too):

```suggestion
  "data": [{"wildcard":"phystrax_hrm.?.csv"}]
```
(or whatever you'd like to call the file)

---

Okay, I can do that. Thanks!

---

Cool, I've matched the ones in other sources and merged - thanks for the app!